### ISSUE TITLE
Idea: add demo folder

### ISSUE THREAD

**Issue by mindware** [sroze/ngInfiniteScroll#159]
A demos folder should be added in the repo or in a related repo.  

The code for the demos listed in the following url unfortunately have their javascript minified, and since the code is not posted elsewhere, this isn't particularly friendly to newcomers. 
http://binarymuse.github.io/ngInfiniteScroll/demos.html

**Comment by sroze** [comment]
You're totally right ! I won't be able to do that before few weeks, so PR are welcome :)
